### ISSUE TITLE
undefined method `id' for nil:NilClass

### ISSUE THREAD

**Issue by maxgronlund** [charliesome/better_errors#305]
Extracted source (around line #127):
better_errors (2.1.1) lib/better_errors/middleware.rb:127:in `internal_call'

```
def internal_call(env, opts) <<<<<<<<<<<<<<<<<<<<<<<<<<<<< line 127
  if opts[:id] != @error_page.id
    return [200, { "Content-Type" => "text/plain; charset=utf-8" }, [JSON.dump(error: "Session expired")]]
  end
```

**Comment by sivunestori** [comment]
I'm facing the same issue. @error_page is nil. Tried with versions 1.1.0, 2.1.0 and 2.2.0. Issue #301 seems to be related.

**Comment by techsethi** [comment]
Having the same issue. Using better_errors with middleman.

**Comment by bfaloona** [comment]
Is this a duplicate of issue #301 ?

**Comment by grekko** [comment]
@techsethi I may have a fix, see my comment in #301

**Comment by RobinDaugherty** [comment]
Duplicate of #301.
